### PR TITLE
Artifact studies

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary

SUMMARY: Content "Microlabs can contain artifacts in containment."

#### Purpose of change

Make use of the new artifact jsonization, and add aditional effects and items

#### Describe the solution

Microlabs can contain a sealed chamber that stores a random "altered object" artifact. `Altered objects` unlike the rest of the artifacts, look like slightly off versions of everyday items, and presumambly are little bits of nether material that have imprinted to our dimension. They mostly have active abilities for now, but I plan to add some new passive effects in the near future

#### Describe alternatives you've considered

Just spawning other existing artifacts within the room.

#### Testing

Game loads

#### Additional context

Mostly did the gameplay affecting parts in here. I plan to follow this pr with a second one adding some lore on the artifacts being studied, and with some new tools placed around the isolation chamber just for fluff.

### CODE REVIEW

File: data/json/artifact/altered_object_active.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    "effect_str": "darkness",
+    "min_duration": 2400,
+    "max_duration": 2400
+  },
+  {
+    "type": "SPELL",
+    "id": "AO_LIFE_DRAIN",
+    "name": { "str": "Artifact Hurtall" },
+    "description": "Hurts you for random damage",
+    "message": "You feel hollow inside",
```
Comment (matskuman5) [inline data/json/artifact/altered_object_active.json:109]:
```suggestion
    "message": "You feel hollow inside.",
```

File: data/json/artifact/altered_object_active.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    "max_aoe": 5,
+    "flags": [ "RANDOM_DAMAGE", "HOSTILE_50" ],
+    "min_duration": 40000,
+    "max_duration": 40000
+  },
+  {
+    "type": "SPELL",
+    "id": "AO_CALL_OF_TINDALOS",
+    "name": { "str": "Artifact Call of Tindalos" },
+    "description": "Summon  hostile hounds of tindalos",
```
Comment (Qrox) [inline data/json/artifact/altered_object_active.json:24]:
```suggestion
    "description": "Summon hostile hounds of tindalos",
```

File: data/json/items/relics/altered_items.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    "price": 1000,
+    "price_postapoc": 10
+  },
+  {
+    "type": "GENERIC",
+    "id": "altered_stopwatch",
+    "symbol": ",",
+    "color": "blue",
+    "looks_like": "wristwatch",
+    "name": { "str": "glitching stopwatch" },
```
Comment (Qrox) [inline data/json/items/relics/altered_items.json:74]:
```suggestion
    "name": { "str": "glitching stopwatch", "str_pl": "glitching stopwatches" },
```

File: data/json/artifact/altered_object_active.json [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    "flags": [ "HOSTILE_SUMMON" ]
+  },
+  {
+    "id": "AO_FORCE_PULL",
+    "type": "SPELL",
+    "name": "Artifact Force Pull",
+    "description": "Pull objects.",
+    "effect": "area_pull",
+    "effect_filter": [ "ally", "hostile", "item", "fd_fire", "fd_blood" ],
+    "effect_iterations": 9,
```
Comment (ZhilkinSerg) [inline data/json/artifact/altered_object_active.json:45]: What's that?

Comment (John-Candlebury) [reply data/json/artifact/altered_object_active.json:45]: Its from  #31783. But there are no docs related to it.

I just copied the json object from there, and checked that the spell worked in-game. Never verified what this entry does.

Comment (KorGgenT) [reply data/json/artifact/altered_object_active.json:45]: the CI is saying it isn't reading it, and magic doesn't have any loading code for it.

**Comment by Mister-Cultist** [comment]
I think I found two but neither had any effect or active.

**Comment by John-Candlebury** [comment]
That is very weird, did you do a clean install of the game?

**Comment by KorGgenT** [comment]
there was a build or two where i accidentally deleted the code that read the json for the "artifact" field in itemgroups.

**Pull request merged**